hatchet/readers/hpctoolkit_reader_latest.py

```diff
@@ -0,0 +1,421 @@
+import os
```

Comment:
Missing license header

Reply:
@ilumsden check my latest commit, this is fixed